### ISSUE TITLE
[Microsoft Teams] Find Chat command fails to load

### ISSUE THREAD

**Issue by pindab0ter** [raycast/extensions#5997]
### Extension

https://www.raycast.com/sven/microsoft-teams

### Description

Instead of seeing the list of available chats, I'm greeted with the "Something went wrong" page.

This is the error that I copied:

```
TypeError: Cannot read properties of null (reading 'createdDateTime')

h5:findChat.tsx:34:25
Or:index.js:5:5486
    at ray-list
lt:index.js:23:19621
```

### Steps To Reproduce

1. Follow the setup instructions in the readme, including the creation of the App registration in the Azure Portal, complete with the granting of the required permissions.
2. Open the Find Chat command and log in successfully
3. Continue to the Find Chat command

**Comment by svenwiegand** [comment]
Thanks for the report @pindab0ter. Are the other features of the extension (setting the presence) work?

**Comment by pindab0ter** [comment]
I can confirm that setting my status does work.

**Comment by svenwiegand** [comment]
Sorry for the long delay @pindab0ter, I was a bit busy. Can you please send a screenshot from your app registration's **API permissions** page in the Azure AD? I guess we have problems here with the permissions. 

The page should look like this:
![CleanShot 2023-04-30 at 11 52 46](https://user-images.githubusercontent.com/2285034/235346966-2a0f39a6-23af-4079-bc51-1433bb53c5c7.png)

**Comment by pindab0ter** [comment]
This is what I have:

<img width="1044" alt="image" src="https://user-images.githubusercontent.com/5128166/236617761-881f6d36-e885-4a96-b989-47959697054c.png">

That's everything in the list and a few other things as well.

**Comment by pindab0ter** [comment]
Sounds good! Looking forward to trying it out!

I can try to run a development build if you like, to see if it fixes the issue.

**Comment by svenwiegand** [comment]
Fixed this in pull request #6603 

Unfortunately the find chat command still isn’t finding all chats. Working on this.

**Comment by svenwiegand** [comment]
Thanks for the confirmation @pindab0ter
